### PR TITLE
FCCEUD-41

### PR DESCRIPTION
**Pull request by adahms**
Applied editing feedback to sections of the end-user documentation.

### CODE REVIEW

**Comment by adahms** [comment]
No changes made to technical content; editorial changes only.

Skipping tech review and passing to @stoobie for peer review.

Steve - can you run through the changes and let me know what you think?

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc
Comment (stoobie) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc:5]: I'm fine with "must", but how do you decide on "must" vs "need to"?

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc
Comment (stoobie) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc:7]: One of the first tech writing guidelines I learned was: Don't use "allows", Do use "enables".

"Enables" is more active and sounds more confident and purposeful than "allows".

I'd suggest: `enables the following actions:`

And if you go with this suggestion, then I think that the actions should start with gerunds (importing, uploading).

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_prerequisites.adoc
Comment (stoobie) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_prerequisites.adoc:21]: I think that this was written before we were told that there is no uploader tool anymore. @benradey @emarcusRH ?

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_prerequisites.adoc
Comment (stoobie) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_prerequisites.adoc:26]: I'm curious if this is still true. @benradey ?

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_prerequisites.adoc
Comment (stoobie) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_prerequisites.adoc:27]: CSB is true, but it's specific to Red Hat. Maybe we should create ifdefs for RH/community?

Should change this to RHEL 7 CSB or later.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_prerequisites.adoc
Comment (stoobie) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_prerequisites.adoc:34]: @benradey Is this still true?

**Review by stoobie — CHANGES_REQUESTED** [review]

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc:2]: @stoobie - this was the change suggested in FCCEUD-31 because it's what we had at the time, but both those attributes are the same today. As far as I can see, 'ProductShortName' is the one used in all our modules except the one about products, so at least today, my recommendation is for us to use the same one.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_prerequisites.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_prerequisites.adoc:21]: Good idea asking the questions from here down.

Once we answer them, let's raise a new issue to make these changes so that we can focus on editorial aspects in this PR.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc:19]: Agreed; will do.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc:11]: Roger that.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc:9]: Agreed; will do.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc:7]: Agreed re. 'enables' - the only question I'd raise here is that if we remove 'you to perform', it becomes unclear who is doing what. Pantheon doesn't do this; the user does, and this is the vehicle that makes it possible for the user to do so.

Would you have any issues with keeping that in?

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc]: Updated.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc]: Updated.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc]: Updated.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc]: Updated.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc]: Updated.

**Comment by adahms** [comment]
Hey @stoobie - pushed some updates to address your questions and add the abstract tag discussed. You've got some open questions - are there any there you'd like set before merging this?

My recommendation with regards to the technical ones is to merge the editorial updates now, ask engineering, and raise an issue to address those when needed.

Until this PR is merged, we'll be blocked on one or two assemblies that require the changes made here.

**Review by stoobie — CHANGES_REQUESTED** [review]

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_products.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_products.adoc:2]: Thanks, @stoobie - updated!

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_products.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_products.adoc]: Good idea, @stoobie - updated!

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_products.adoc
Comment (adahms) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_products.adoc]: Changed to a comma.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc
Comment (stoobie) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc]: No, that's fine, I see what you mean.

**Review by stoobie — APPROVED** [review]
@adahms  Going with your suggestion to move technical questions to another PR.

**Comment by adahms** [comment]
Thanks, @stoobie!

@redhataccess/pantheon-developers - ready for merge.

**Comment by stoobie** [comment]
Approved.

**Review by stoobie — APPROVED** [review]

**Review by benradey — APPROVED** [review]

**Pull request merged**
